src/components/MainArea.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Column from './Column';
+import EditableText from './EditableText';
+import {setBoardName} from '../store';
+
+const MainArea = ({
+  name,
+  setBoardName
+}) => {
+  return (
+    <main>
+      <div className="main-menu">
+        <EditableText
+          id="board-name"
+          name="board-name"
+          value={name}
+          onChange={setBoardName}
+        />
+      </div>
+      <div className="main-area-content">
+        <Column
+          title="To Do"
+          cards={[
+            { title: 'Hallo', id: 0 },
+            { title: 'Hola', id: 1 },
+            { title: 'Gulp', id: 2 }
+          ]}
+        />
+        <Column
+          title="In Progress"
+          cards={[
+            { title: 'Start', id: 3 },
+            { title: 'Stop', id: 4 }
+          ]}
+        />
+        <Column
+          title="Done"
+          cards={[
+            { title: 'Tada', id: 5 }
+          ]}
+        />
+      </div>
+    </main>
+  );
+};
+
+MainArea.propTypes = {
+  name: PropTypes.string.isRequired,
+  setBoardName: PropTypes.func.isRequired
+};
+
+const mapStateToProps = state => {
+  return {name: state.name};
```

Comment:
[formatting] Pad the object with whitespaces like `{ name: state.name }` for consistency with the rest of the code.